vpr/src/route/rr_graph.cpp

```diff
@@ -2501,6 +2501,70 @@ static vtr::NdMatrix<std::vector<int>, 4> alloc_and_load_track_to_pin_lookup(vtr
     return track_to_pin_lookup;
 }
 
+/* Prints all the data about node inode to file fp.                    */
+void print_rr_node(FILE* fp, const std::vector<t_rr_node>& L_rr_node, int inode) {
```

Comment:
I think there might be a function like this somewhere in the code already? vpr is already produces an `rr_graph.echo` file which looks like;
```
RR node: 0 type: SOURCE location: (0,1) class: 0
1 edge(s): 1
Switch types: 0
Capacity: 1
R: 0  C: 0
Cost_index: 0

RR node: 1 type: OPIN location: (0,1) pin: 0 pin_name: IBUF.I[0]
2 edge(s): 38 39
Switch types: 1 1
Capacity: 1
R: 0  C: 0
Cost_index: 2
```